README.md

```diff
@@ -34,21 +34,71 @@ Refer to these files by adding a `<link>` and a `<script>` element into your HTM
 
 ### Install with NPM
 
-If you have `node` installed on your machine, you can use `npm` to install the Web Design Standards. In your `package.json` list the following under `dependencies`:
+If you have `node` installed on your machine, you can use `npm` to install the
+Web Design Standards.
 
-```json
-  "uswds": "git@github.com:18F/web-design-standards.git#v0.9.0"
+```
+cd path/to/project-using-npm
+npm install --save uswds
 ```
 
-On subsequent runs of `npm install`, the package will be installed in `node_modules` under `uswds`. The CSS will be *not* compiled after it is installed. Instead, you are able to compile it however it fits into your project.
+This will add the Web Design Standards as a dependency for your project and save
+it to your `package.json` file. The package will be installed in `node_modules`
+under `uswds`. The CSS file(s) are generated on the `npm-prepublish` hook and can
+found in the following directory: `./node_modules/uswds/assets/css`.
 
-An `npm` task called `build-sass` is supplied, though you are not required to use it, that will compile the Sass to a CSS file named `uswds.css` (in `node_modules/uswds/assets/css/uswds.css`).
+An `npm` task called `build-sass` is supplied, though you are not required to
+use it, that will compile the Sass to a CSS file named `uswds.css` (in
+`node_modules/uswds/assets/css/uswds.css`).
 
-If you simply want to compile the Sass you could do something like:
+Note: You might get an [`npm` warning related to `lodash`](https://github.com/18F/web-design-standards/pull/902#issuecomment-161076213), but you can generally ignore it.
```

Comment:
I like this idea of saying this. It can be scary to get warnings, although they are common in node installs.